### DISCUSSION TITLE
Please make pylance available from open-vsx.org

### DISCUSSION THREAD

**Discussion by mikekanenz** [microsoft/pylance-release#4519] [Enhancement]
Please add a copy of the extension to open-vsx.org so it can be used in code-server as obtaining the extension from marketplace is not allowed from code-server.

**Comment by debonte** [comment]
Pylance cannot be made available on open-vsx.org because it is not open-source. However, our open-source language server (Pyright) is [available there](https://open-vsx.org/?search=pyright&category=&sortBy=relevance&sortOrder=desc).

FYI, if you're looking for a web editing experience that supports Pylance, it is available for use in https://vscode.dev.
